casper/src/test/scala/coop/rchain/casper/genesis/contracts/TestUtil.scala

```diff
@@ -65,7 +65,7 @@ object TestUtil {
         genesisPk = Secp256k1.newKeyPair._2,
         vaults = Vault(
           RevAddress.fromPublicKey(Secp256k1.toPublic(ConstructDeploy.defaultSec)).get,
-          900000
+          9000000
```

Comment:
Aren't these two vaults the same? We should have a constant "defaultVault" instead of keeping the magic number in two places

Reply:
Even better, I'm going to rip the shreds out of Testutil and MultiParentCasperTestUtil in the next PR, replacing them with a single GenesisBuilder. How about that? :)